test/views/NotificationsViewSpec.scala

```diff
@@ -23,14 +23,15 @@ import controllers.routes
 import models.declaration.notifications.Notification
 import models.declaration.submissions.{Action, RequestType, Submission, SubmissionStatus}
 import org.jsoup.nodes.Document
-import views.declaration.spec.AppViewSpec
+import unit.tools.Stubs
+import views.declaration.spec.UnitViewSpec
 import views.html.submission_notifications
 import views.tags.ViewTest
 
 @ViewTest
-class NotificationsViewSpec extends AppViewSpec {
+class NotificationsViewSpec extends UnitViewSpec with Stubs {
```

Comment:
Simple technical stuff - maybe would be worth to move `mainTemplate` from `Stubs` to `UnitViewSpec`

Reply:
But, we're using `mainTemplate` in `ControllerSpec` too, so our ControllerSpec must extends `UnitViewSpec`, so we're breaking those names `View` -> `Controller` and we'll be back to the some kind old structure, because it'll be a mess to extends `ViewSpec` in `ControllerSpec`

Reply:
Why ControllerSpec require to have `mainTemplate` ? We can not test them without dependencies to vies ?

Reply:
Because in few ControllerSpec we're not mocking the view, but we're creating a page. To create every page in the whole application you need to pass mainTemplate.

Reply:
Possibly we can go as it is and once we have some work on the Controllers we can refactor this.